modules/displayPokemon.js

```diff
@@ -0,0 +1,78 @@
+import { getComments } from './involvementRequests.js';
+import displayMessage from './displayMessage.js';
+/* eslint-disable no-new */
+const cardPoke = document.querySelector('.cardModal');
+const $grafica = document.querySelector('#grafica');
+
+const displayPokemon = async (card, index) => {
+  let nroAbility = 0;
+  let datosIngresos = {};
+  const etiquetas = [];
+  const datas = [];
+  // eslint-disable-next-line no-undef
+  let pokeinfo = '';
+  pokeinfo += `
+    
+    <img src="${card[index].data.sprites.other['official-artwork'].front_default}" alt="pokemon">
+    <div id="info">
+    <h2>${card[index].data.name.toUpperCase()}</h2>
+    `;
+  // eslint-disable-next-line no-restricted-syntax
+  for (const abilityOne of card[index].data.abilities) {
+    pokeinfo += `
+        <p>Ability ${nroAbility + 1}:  ${abilityOne.ability.name}</p>`;
+    // eslint-disable-next-line no-new, no-undef
+    nroAbility += 1;
+  }
+  cardPoke.innerHTML = `${pokeinfo}</div>`;
+  // eslint-disable-next-line no-restricted-syntax
+  for (const pokemonStat of card[index].data.stats) {
+    etiquetas.push(pokemonStat.stat.name);
+    datas.push(pokemonStat.base_stat);
+  }
+  datosIngresos = {
+    label: 'Stats',
+    data: datas,
+    order: 1,
+    backgroundColor: [
+      'rgba(163,221,203,0.2)',
+      'rgba(232,233,161,0.2)',
+      'rgba(230,181,102,0.2)',
+      'rgba(229,112,126,0.2)',
+    ], // Color de fondo
+    borderColor: [
+      'rgba(163,221,203,1)',
+      'rgba(232,233,161,1)',
+      'rgba(230,181,102,1)',
+      'rgba(229,112,126,1)',
+    ], // Color del borde
+    borderWidth: 3, // Ancho del borde
+  };
+  const options = {
+    responsive: false,
+    devicePixelRatio: 1,
+    maintainAspectRatio: true,
+    scale: {
+      ticks: {
+        beginAtZero: true,
+        max: 200,
+      },
+    },
+  };
+  // eslint-disable-next-line no-new, no-undef
```

Comment:
Please don't disable the linters. Consider fixing the linter issues instead of disabling the linters.

Reply:
As requested by the previous code reviewers please kindly enable eslint in all files were it has been disabled. linters help us write clean code and it is not allowed to disable them